app/src/main/res/values/strings.xml

```diff
@@ -1483,7 +1483,7 @@
     <!-- Preference option for never saving passwords in Fenix -->
     <string name="preferences_passwords_save_logins_never_save">Never save</string>
     <!-- Preference for autofilling saved logins in Fenix -->
-    <string name="preferences_passwords_autofill" tools:ignore="UnusedResources">Autofill</string>
+    <string name="preferences_passwords_autofill" moz:removedIn="93" tools:ignore="UnusedResources">Autofill</string>
```

Comment:
will strings ever be removed in patches and/or dot-version releases?

Reply:
Ideally our automatization should take care of when we do release, hopefully we can integrate it to our release process.

Reply:
Sorry, I was more asking if we have ever had patch releases that removed strings, or if that's typically reserved for major version releases.

Reply:
I think ideally we would like to run removing string tool for major release to make sure the removed strings are not longer used as having minor and bug fixed versioning could bring more complexity.

Reply:
At least, this what see initially, do you have any suggestions ? :)